packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/google-drive.scss

```diff
@@ -20,3 +20,6 @@
         }
     }
 }
+.picker-dialog {
+    z-index: 9999 !important;
```

Comment:
Do we need to add important here?

Reply:
@niloofar-deriv yes we needed this line because it was a legacy change and we had 2 duplicate components for google drive and the only line difference was this. So i kept it there maybe it was added to fix some issue and discarded the one component.

Reply:
Could u please check it with your team if we don't need it please remove it :pray:

Reply:
@niloofar-deriv found the it was give to give the modal more precedence then the load dialoagt 
![image](https://github.com/binary-com/deriv-app/assets/97010868/80a5f47f-2e92-49c0-9b3c-166feee46879)
so we will need it here also i think we dont require this high z-index value will change this

Reply:
Oh ok then, thanks for checking :pray: